src/dbnode/integration/setup.go

```diff
@@ -160,14 +162,24 @@ type TestSetup interface {
 
 type storageOption func(storage.Options) storage.Options
 
+// NewTestSetup returns a new test setup for non-dockerized integration tests.
+func NewTestSetup(
```

Comment:
nit: just export `newTestSetup` below?

Reply:
Ah, I added this since there's a bunch of refactoring work needed to get `*testSetup` -> `TestSetup` and I figured we could punt on that for now.

Reply:
Ah, you know what I just ended up migrating everything over haha.